### PR TITLE
[T4A5][W11-B3]Wong Kang Fei

### PR DESCRIPTION
**Pull request by kfwong**

### CODE REVIEW

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package seedu.addressbook.data.tag;
+
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/tag/Tagging.java]: Forgot to put the header comment.

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import seedu.addressbook.data.person.Person;
+
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/tag/Tagging.java]: Forgot to put the header comment.

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    public Tag getTag() {
+        return tag;
+    }
+
+    public boolean isDeleted(){
+        return isDeleted;
+    }
+    
+    public void setDeleted(boolean isDeleted){
+        this.isDeleted = isDeleted;
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/tag/Tagging.java]: You can also consider setting this in the constructor.

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    }
+    
+    @Override
+    public String toString(){
+        String deletedSign = isDeleted? IS_DELETED: IS_NOT_DELETED;
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/tag/Tagging.java]: You ternary expression doesn't follow our coding convention, which is `alpha = (aLongBooleanExpression) ? beta : gamma;`

**Comment by jeffryhartanto** [comment]
Some comments added. Please close the PR after reading comments.

File: src/seedu/addressbook/data/tag/Tagging.java (hunk 4 above)
Comment (kfwong) [reply src/seedu/addressbook/data/tag/Tagging.java:37]: Ahh right! thank you for the correction!

**Pull request closed without merging**